storage/backend/azure/azure.go

```diff
@@ -112,17 +113,15 @@ func (b *Backend) Get(ctx context.Context, p string, w io.Writer) (err error) {
 func (b *Backend) Put(ctx context.Context, p string, r io.Reader) error {
 	b.logger.Log("msg", "uploading the file with blob", "name", p)
 
-	blobURL := b.containerURL.NewBlockBlobURL(p)
-	if _, err := azblob.UploadStreamToBlockBlob(ctx, r, blobURL,
-		azblob.UploadStreamToBlockBlobOptions{
-			BufferSize: defaultBufferSize,
-			MaxBuffers: defaultMaxBuffers,
-		},
-	); err != nil {
-		return fmt.Errorf("put the object, %w", err)
+	m, err := ioutil.ReadAll(r)
```

Comment:
The whole idea behind `v1.1.0` release to get rid offer buffering into memory.
I understand you have practical needs for this, however I'd rather fix the underlying issue with streaming API rather than falling back to this.

I need to spend some time on the underlying issue. Do you have any pointers? Documentation etc.

Reply:
It was a quick solution. I couldn't find an easy approach to use streaming api to solve this. I'll try to find out if it's supported by azure highlevel api.